pyproject.toml

```diff
@@ -20,6 +20,7 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
+pydantic = "^1.5.1"
```

Comment:
Ouch, depending on pydantic for just one annotation in the code :sweat_smile: Could we avoid this and put it in dev deps instead?